docs/source/logging/index.md

```diff
@@ -6,7 +6,7 @@ Kedro uses [Python's `logging` library](https://docs.python.org/3/library/loggin
 2. Your project-side logging configuration. Every project generated using Kedro's CLI `kedro new` command includes a file `conf/base/logging.yml`. You can alter this configuration and provide different configurations for different run environment according to the [standard Kedro mechanism for handling configuration](../configuration/configuration_basics.md).
 
 ```{note}
-Providing project-side logging configuration is entirely optional. You can delete the `conf/base/logging.yml` file and Kedro will run using the framework's built in configuration.
+Providing project-side logging configuration is entirely optional. You can delete the `conf/logging.yml` file and Kedro will run using the framework's built in configuration.
```

Comment:
The logging.yml file won't be shipped at all with a project template after this change is completed, so this line in the docs won't make sense.

I think the whole logging docs will need to be rewritten tbh, since it all changes a lot - I would probably save that for a separate PR, because it's a big change.